earthquakes/notebooks/Earthquakes.ipynb

```diff
@@ -0,0 +1,655 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "sys.path.append(\"..\")\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "pd.options.display.precision = 15\n",
+    "data_dir = \"D:/KaggleData/earthquakes/\"\n",
```

Comment:
I guess it is wise to keep the same structure of our folder on Github as in other competitions (so that everybody can run the code from their local machine). Therefore I would suggest to add:
import sys
sys.path.append("../..") 
data_dir = "../data/"

Reply:
Right, I had it this way because my C: drive is pretty much full and the data is quite large (~9GB).